dali/test/python/test_RN50_data_fw_iterators.py

```diff
@@ -22,6 +22,7 @@
 import tensorflow as tf
 from nvidia.dali.plugin.mxnet import DALIClassificationIterator as MXNetIterator
 from nvidia.dali.plugin.pytorch import DALIClassificationIterator as PyTorchIterator
+from nvidia.dali.plugin.paddle import DALIClassificationIterator as PaddleIterator
```

Comment:
you need to add `paddle` to `pip_packages` variable in `qa/TL0_FW_iterators/test.sh` `qa/TL1_separate_executor/test.sh` and `qa/TL2_FW_iterators_perf/test.sh`

E.g.: change
```
pip_packages="nose numpy opencv-python tensorflow-gpu torch torchvision mxnet-cu{cuda_v}"
```
to 
```
pip_packages="nose numpy opencv-python tensorflow-gpu torch torchvision mxnet-cu{cuda_v} paddle"
```

Reply:
Done